packages/core/src/Provider/types/StakePool.ts

```diff
@@ -0,0 +1,136 @@
+import { Address, PoolId, Hash16 } from '@cardano-ogmios/schema';
+import { ExtendedStakePoolMetadata } from './ExtendedStakePoolMetadata';
+import { Lovelace } from '../../Ogmios';
+
+/**
+ * Within range [0; 1]
+ */
+export type Percent = number;
+
+export type TransactionId = Hash16;
+
+/**
+ * https://github.com/cardano-foundation/CIPs/blob/master/CIP-0006/CIP-0006.md#on-chain-referenced-main-metadata-file
+ */
+export interface StakePoolMetadata {
+  /**
+   * Pool ticker. uppercase
+   * 5 Characters Maximum, Uppercase letters and numbers
+   */
+  ticker: string;
+  /**
+   * A name for the pool
+   * 50 Characters Maximum
+   */
+  name: string;
+  /**
+   * Text that describes the pool
+   * 50 Characters Maximum
+   */
+  description: string;
+  /**
+   * A website URL for the pool
+   * 64 Characters Maximum, must be a valid URL
+   */
+  homepage: string;
+  /**
+   * A URL for extended metadata
+   * 128 Characters Maximum, must be a valid URL
+   */
+  extDataUrl?: string;
+  /**
+   * A URL with the extended metadata signature
+   * 128 Characters Maximum, must be a valid URL
+   */
+  extSigUrl?: string;
+  /**
+   * the public Key for verification
+   * optional, 68 Characters
+   */
+  extVkey?: Hash16; // Review: is this the correct type alias?
+  /**
+   * Extended metadata defined by CIP-6
+   */
+  ext?: ExtendedStakePoolMetadata;
+}
+
+export interface StakePool {
+  /**
+   * Stake pool ID as a bech32 string
+   */
+  id: PoolId;
+  /**
+   * Stake pool ID as a hex string
+   */
+  hexId: Hash16;
+  /**
+   * Stake pool's pledge
+   */
+  pledge: {
+    /**
+     * Stake pool's declared pledge
+     */
+    declared: Lovelace;
+    /**
+     * Stake pool's currently live pledge
+     */
+    live: Lovelace;
+  };
+  /**
+   * Stake pool's running cost
+   */
+  cost: {
+    /**
+     * Stake pool's fixed cost
+     */
+    fixed: Lovelace;
+    /**
+     * Stake pool's fixed cost
+     */
+    margin: Percent;
+  };
+  /**
+   * Stake pool stats
+   */
+  stats: {
+    /**
+     * Total blocks minted by the pool
```

Comment:
```suggestion
     * Total blocks created by the pool
```
We're avoiding the terminology clash with _minting tokens_, and the negative connotation surrounding _forging_